roles/github/templates/build-kayobe-docker-image.yml.j2

```diff
@@ -1,3 +1,9 @@
+<%- if github_environment_selector == 'input' -%>
+<%- set github_runs_on = github_runs_on + ['${{ matrix.environment }}'] -%>
+<%- set _ = github_default_registry.update({"url": "${{ vars[format('{0}_REGISTRY_URL', matrix.environment)] }}" }) -%>
+<%- set _ = github_default_registry.update({"username": "${{ vars[format('{0}_REGISTRY_USERNAME', matrix.environment)] }}" }) -%>
+<%- set _ = github_default_registry.update({"password": "${{ secrets[format('{0}_REGISTRY_PASSWORD', matrix.environment)] }}" }) -%>
```

Comment:
Need to set KAYOBE_ENVIRONMENT when selector is single and image is not shared?

Reply:
I don't believe so if there is a single environment or no environment then the matrix isn't not used and that is the only place anything environment related is used within this workflow